default.xml

```diff
@@ -64,4 +64,6 @@
   <project remote="github" name="madler/zlib" path="zlib" revision="refs/tags/v1.3.1" />
 
   <project remote="github" name="ninja-build/ninja" path="ninja" groups="notdefault,dependencies" revision="master" />
+
+  <project remote="github" name="microsoft/mimalloc" path="mimalloc" revision="refs/tags/v2.1.7" />
```

Comment:
The other vendors are alphabetical, can we hoist this above `ninja-build`?

Reply:
Done